### PR TITLE
Allow vrrp & icmp on provisioning network

### PR DESCRIPTION
**Pull request by kashifest**
Add firewall rules to allow icmp and vrrp packets on provisioning network. 

Co-authored-by: maelk <mael.kimmerlin@est.tech>, Xenwar <anwar.hassen@est.tech>

### CODE REVIEW

File: vm-setup/roles/firewall/defaults/main.yml [hunk 1]
```diff
@@ -17,3 +17,6 @@ pxe_udp_ports:
   - "67"
   - "68"
   - "69"
+ironic_keepalived_proto:
+  - "112"
+  - "icmp
```
Comment (maelk) [inline vm-setup/roles/firewall/defaults/main.yml]: missing a quote at the end of the line

**Comment by maelk** [comment]
the CI failed because of the typo

File: vm-setup/roles/firewall/defaults/main.yml (hunk 1 above)
Comment (kashifest) [reply vm-setup/roles/firewall/defaults/main.yml]: yeah, noticed that. Updated

**Comment by kashifest** [comment]
> the CI failed because of the typo
CI passed now

**Comment by maelk** [comment]
Please rebase and run the tests again

**Comment by kashifest** [comment]
> Please rebase and run the tests again

Rebased and ran the tests

**Pull request merged**